radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt

```diff
@@ -98,12 +108,17 @@ class RadialGraph : ConstraintLayout {
             )
         } else {
             if (graphNode != GraphNode.NONE) {
-                Log.w(TAG, "No value passed for the `app:graphNodeColor` attribute. It will default to Magenta")
+                Log.w(
+                    TAG,
+                    "No value passed for the `app:graphNodeColor` attribute. It will default to Magenta"
+                )
             }
 
             ContextCompat.getColor(context, R.color.node_defaultColor)
         }
 
+        val graphNodeIcon = attributes.getDrawable(R.styleable.RadialGraph_graphNodeIcon)
```

Comment:
Can we add a warning similar to the one above to check if the NodeType is set to `ICON` and no drawable was passed?

Reply:
Should we perhaps provide a default icon as well?

Reply:
If a icon is not provided I think an error should be thrown.